vendor/cookbooks/rails/recipes/default.rb

```diff
@@ -33,6 +33,7 @@
 
   # Include library helpers
   ::Chef::Resource.send(:include, Rails::Helpers)
+  ::Chef::Recipe.send(:include, Rails::Helpers)
```

Comment:
without this line, the following wouldn't work, method missing:
```
98: ssl_certificate_path = ssl_certificate(applications_root, app, app_info)
```
since it's a level of recipe, not resource such as:
```
cookbook_file pathname.to_s do
  ssl_certificate_path = ssl_certificate(applications_root, app, app_info)
end
```